### PR TITLE
chore(build): enable Gradle wrapper validation (GitHub Actions)

### PR DESCRIPTION
**Pull request by sullis**
https://blog.gradle.org/gradle-wrapper-checksum-verification-github-action

### CODE REVIEW

**Comment by sullis** [comment]
@fieldju

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: gradle/wrapper-validation-action@v1
```
Comment (jan25) [inline .github/workflows/build.yml:19]: Using name field would be nice. wdyt?
```
steps:
  name: Validation step
  uses: ...
```
Same for other changes :)

Comment (sullis) [reply .github/workflows/build.yml]: Fixed.

**Comment by sullis** [comment]
WDYT? @jan25

**Comment by jan25** [comment]
Thanks @sullis. LGTM. 

Rest of yaml can follow similar form too. Probably can go in separate PR.

**Pull request merged**